docker-compose.yml

```
@@ -77,3 +78,18 @@ services:
     volumes:
       - ./var/redis/dаta:/root/redis
       - ./var/redis/redis.conf:/usr/local/etc/redis/redis.conf
+
+  celery:
+    image: vitrina-app:latest
+    container_name: vitrina-celery
+    command: celery -A vitrina worker -l info
+    environment:
+      - REDIS_URL=redis://redis:6379/
+      - DJANGO_SETTINGS_MODULE=vitrina.settings
+    volumes:
+      - .:/app
+    depends_on:
+      - redis
+      - vitrina
```

Technically it does not depend and celery container should run (and process tasks) even if main web-app is down.
```suggestion
      - redis
```